java/casting.md

```diff
@@ -8,4 +8,10 @@ Mit einem cast lässt sich nicht jedes Objekt X in ein Objekt Y umwandeln. Es ka
 
 Beispielsweise kann man einen `CommandSender` zu `Player` casten, weil `Player` von `CommandSender` erbt. Es gibt aber noch andere `CommandSender` - z.B. die Konsole oder Command Blocks - deshalb sollte man vorher mit `instanceof` überprüfen, ob es sich wirklich um einen `Player` handelt.
 
+Das in Java 14 eingeführte `Pattern Matching` ist dabei sehr nützlich - das führt zu simplerem und einfacherem Code.
+```java
+if (sender instanceof Player player) 
+```
+Ohne eine neue Variable zu defnieren, ist es möglich `player` zu verwenden.
```

Comment:
Well, actually "player" is just a local variable. You should mention more, that you don't need a explicit cast.